echopype/echodata/sensor_ep_version_mapping/ep_version_mapper.py

```diff
@@ -0,0 +1,30 @@
+from _echopype_version import version as __version__  # noqa
+
+from .v05x_to_v06x import convert_v05x_to_v06x
+
+
+def map_ep_version(echodata_obj):
```

Comment:
I like how you have made this flexible for expansion later!  :D
(Not that we want a lot of these...)